package.json

```diff
@@ -33,7 +33,8 @@
   ],
   "homepage": "https://github.com/bithost-gmbh/ngx-mat-select-search",
   "peerDependencies": {
-    "@angular/material": "^12.0.0 || ^13.0.0 || ^14.0.0"
+    "@angular/material": "^12.0.0 || ^13.0.0 || ^14.0.0",
+    "typescript-formatter": "^7.2.2"
```

Comment:
this package was last published 4y ago (https://www.npmjs.com/package/typescript-formatter) and looks unmaintained. I think we don't want to depend on that one